src/Underscore.php

```diff
@@ -83,9 +119,13 @@ public function __call($method, $args)
      */
     public static function __callStatic($method, $args)
     {
-        $payloadClass = sprintf('\Underscore\Initializer\%sInitializer', ucfirst($method));
-        /** @var $payload callable */
-        $payload = new $payloadClass();
+        $payload = static::fromMixin($method);
```

Comment:
It does, but we need way to distinguish mixin type initializer or mutator (or accessor?).